examples/multi-datacenter/submit_replicator_dc2_to_dc1.sh

```diff
@@ -3,14 +3,16 @@
 HEADER="Content-Type: application/json"
 DATA=$( cat << EOF
 {
-  "name": "replicator-dc2-to-dc1",
+  "name": "replicator-dc2-to-dc1-topic1",
   "config": {
     "connector.class": "io.confluent.connect.replicator.ReplicatorSourceConnector",
     "topic.whitelist": "topic1",
     "key.converter": "io.confluent.connect.replicator.util.ByteArrayConverter",
     "value.converter": "io.confluent.connect.replicator.util.ByteArrayConverter",
```

Comment:
key.converter, value.converter and header.converter are all worker level config. They should be set in docker-compose.yml, not in the connector configuration.

Reply:
They can be set in a worker or overridden by a connector.  Calling this out in the connector for two reasons:

1. draw attention to what these configuration parameters are set to for Replicator
2. allows connectors with different settings to co-exist on the same cluster of workers